src/vs/workbench/contrib/mcp/browser/mcpWorkbenchService.ts

```diff
@@ -302,6 +319,29 @@ export class McpWorkbenchService extends Disposable implements IMcpWorkbenchServ
 		return undefined;
 	}
 
+	async handleURL(uri: URI): Promise<boolean> {
+		if (uri.path !== 'mcp/install') {
+			return false;
+		}
+
+		let parsed: IMcpServerConfiguration & { name: string };
```

Comment:
The server configuration given in this URL can also define `inputs` ([e.g.](https://github.com/github/github-mcp-server?tab=readme-ov-file#local-github-mcp-server)) to be applied into the config.